sdk/springappdiscovery/Azure.ResourceManager.SpringAppDiscovery/tests/Tests/AppCRUDTests.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.SpringAppDiscovery.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.SpringAppDiscovery.Tests.Tests
+{
+    [TestFixture]
+    public class AppCRUDTests : SpringAppDiscoveryManagementTestBase
+    {
+        public AppCRUDTests() : base(true)
```

Comment:
we should keep the `isAsync ` parameter here and pass it into the base ctor.
This is a requirement of the test framework